pytermgui/ansi_interface.py

```diff
@@ -723,114 +809,132 @@ def translate_mouse(code: str, method: str) -> list[MouseEvent | None] | None:
             "34": MouseAction.RIGHT_CLICK,
             "35": MouseAction.RELEASE,
             "64": MouseAction.LEFT_DRAG,
-            "66": MouseAction.RIGHT_CLICK,
+            "66": MouseAction.RIGHT_DRAG,
             "96": MouseAction.SCROLL_UP,
             "97": MouseAction.SCROLL_DOWN,
         },
     }
-
     mapping = mouse_codes[method]
     pattern = mapping["pattern"]
-
     events: list[MouseEvent | None] = []
     for sequence in code.split("\x1b"):
         if len(sequence) == 0:
             continue
-
         matches = list(pattern.finditer(sequence))
         if matches == []:
             return None
-
         for match in matches:
             identifier, *pos, release_code = match.groups()
 
             # decimal_xterm uses the last character's
             # capitalization to signify press/release state
             if len(release_code) > 0 and identifier == "0":
                 identifier += release_code
-
             if identifier in mapping:
                 action = mapping[identifier]
                 assert isinstance(action, MouseAction)
-
                 events.append(MouseEvent(action, (int(pos[0]), int(pos[1]))))
                 continue
-
             events.append(None)
-
     return events
 
 
 # shorthand functions
-def print_to(pos: tuple[int, int], *args: Any, **kwargs: Any) -> None:
-    """Print text to given `pos`.
-
-    This passes through all arguments (except for `pos`) to the `print`
-    method."""
+def print_to(
+    pos: tuple[int, int],
+    *objects: Any,
+    sep: str = " ",
+    end: str = " ",
+    file: Optional[IO[str]] = None,
+    flush: bool = True,
+) -> None:
+    """
+    Print text to given `pos`.
 
+    This passes through all arguments (except for `pos`) to the `print` method.
+    """
     move_cursor(pos)
-    print(*args, **kwargs, end="", flush=True)
+    print(*objects, sep=sep, end=end, file=file, flush=flush)
```

Comment:
No need to individually take all `kwargs` of `print` as just unpacking `**kwargs` into it does the same thing. People using this relatively low level (in terms of this library) function will more than likely know the signature of `print` anyways, so this is just redundant.

Reply:
I just followed `print`'s signature